packages/world/src/EntityId.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { BaseEntity } from "./codegen/tables/BaseEntity.sol";
+
+type EntityId is bytes32;
+
+function baseEntityId(EntityId self) view returns (EntityId) {
+  EntityId base = BaseEntity._get(self);
```

Comment:
This would fail if it's used in the `experience` namespace right? Since its no longer root?

Reply:
ah you are right, let's figure out first if we want to keep root systems